packages/ifpack2/src/Ifpack2_BlockTriDiContainer_def.hpp

```diff
@@ -115,6 +115,7 @@ namespace Ifpack2 {
     impl_->overlap_communication_and_computation = overlapCommAndComp;
     
     impl_->Z = typename impl_type::tpetra_multivector_type();
+    impl_->W = typename impl_type::tpetra_multivector_type();
```

Comment:
this function is an internal function to initialize variables. Your point is right but I did this  because I have an experience that the default constructor in some compilers does not really initialize its member variables. It is just a safe way to do it myself and does not harm performance.

Reply:
ok